test/lit/passes/outlining.wast

```diff
@@ -505,3 +505,37 @@
     )
   )
 )
+
+;; Test outlining works with call_indirect
+(module
+  (table funcref)
+  (func
+    (call_indirect
+      (param i32)
+      (i32.const 0)
+      (i32.const 0)
```

Comment:
It would be good to add a second argument to the function and make all the operands to the `call_indirect` different to test that the right operand ends up in the right place.

Reply:
done